backend_plugin/src/main/evaluators/Evaluator.java

```diff
@@ -57,22 +68,37 @@ private void initializeListeners(ITextEditor textEditor) {
 		DocumentChangesTracker docTracker = new DocumentChangesTracker(this);
 		doc.addDocumentListener(docTracker);
 		this.documentChangesTracker = docTracker;
+		
+		// Add a AnnotationModelListener to the AnnotationModel
+		AnnotationModelListener annoTracker = new AnnotationModelListener(this);
```

Comment:
We also create a new local variable when we can directly assign the new `AnnotationModelListener` to the instance variable.